.github/workflows/update-refs.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
           ref: main
       - name: Update references
         run: |
-          sed -r -i 's/(ref=.*)/ref='"$GITHUB_SHA"'/g' `grep 'ref=' -rl *`
+          sed -r -i 's/(ref=[a-zA-Z0-9_.-]*)/ref='"$GITHUB_SHA"'/g' `grep 'ref=' -rl *`
```

Comment:
Shouldn't it be just `[a-z0-9]*` for the github refs?

Reply:
Sure! Thanks